### DISCUSSION TITLE
2023-09-07

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#480] [daily_meeting]
### 오늘의 할 일
- 사용자 유치 계획 discussion에 하나씩 아이디어 남기기 (9/8)
- 개인용 서비스를 유지하는 방향에 대해 논의

- 프론트엔드
  - 팀 미션(오늘 오전까지)
  - 리팩토링

### 개인 일정
- 노아
  - 5시 ~ 6시 근로

### 내일의 할 일
- SSE 관련해서 이야기 & 결정

**Comment by nlom0218** [comment]
### 체크인

8점 - 이제 슬슬 테코톡을 준비해야겠어요 🥲

### 체크아웃

9점 - 데이터 패칭 아리송합니다  😭

**Comment by aak2075** [comment]
### 체크인

10점 - 오늘 테코톡 기대되네요

### 체크인

10점 - 리팩토링은 됐는데 더 하고 싶지만 일단 완성하기로...

**Comment by jaehee329** [comment]
### 체크인
3점 - 오늘 테코톡 기대되네요

### 체크아웃
7점 - 테코톡 끝났잖아

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 오늘 늦게까지 자고 밥도 먹고 와서 컨디션은 좀 회복된 것 같습니다. 오늘 할일 좀 잘 처리하면 더할 나위가 없겠네요.

### 체크아웃
7점 - 생각보다 많이 한게 없는 것 같네용...

**Comment by woosung1223** [comment]
### 체크인
10점 - 어제 늦게 자서 원래는 5점 줘야하는데 테코톡 기대되서 10점

### 체크아웃
9점 - 오늘 먼가 많이 한거 같은데 많이 한게 없는 것 같기도 해서 -1점

**Comment by yeopto** [comment]
### 체크인

10점 - 피곤하지만 오늘 하루는 뭘 알아갈지 기대되서 좋네요

### 체크아웃

10점 - 하루가 빨리 갔네요!

**Comment by woo-jk** [comment]
### 체크인
9점 - 어제 좀 일찍자서 덜피곤하다 ㅎㅎ

### 체크아웃
8점 - 고민만 엄청하고 뭔가 진전된게 없는듯하다 ㅜ
